functions/prepare_component.sh

```diff
@@ -763,6 +763,32 @@ prepare_component() {
     fi
   fi
 
+  if [[ "$component" =~ ^(ruffle|all)$ ]]; then
+  component_found="true"
+    if [[ "$action" == "reset" ]]; then # Run reset-only commands
+      log i "----------------------"
+      log i "Preparing Ruffle"
+      log i "----------------------"
+      if [[ $multi_user_mode == "true" ]]; then # Multi-user actions
+        log d "Figure out what Ruffle needs for multi-user"
+      else # Single-user actions
+        # NOTE: the component is writing in "." so it must be placed in the rw filesystem. A symlink of the binary is already placed in /app/bin/
```

Comment:
Is this comment real or a leftover from Vita3K?

Reply:
Yes it's a left over from Vita. Not sure ruffle has a multi user mode so this can go.

Reply:
I was referring to:

```# NOTE: the component is writing in "." so it must be placed in the rw filesystem. A symlink of the binary is already placed in /app/bin/```

The multi user comment #TODO instead must stay there